pkg/cluster/internal/kubeadm/config.go

```diff
@@ -537,6 +539,7 @@ evictionHard:
 {{ range $key := .SortedFeatureGateKeys }}
   "{{ $key }}": {{ index $.FeatureGates $key }}
 {{end}}{{end}}
+localStorageCapacityIsolation: {{if .DisableLocalStorageCapacityIsolation}}false{{else}}true{{end}}
```

Comment:
agree

Reply:
Agree on cleaner but the unknown fields will never reach kubelet, kubeadm will drop any unknown field because it defaults kubelet config itself. We’ve run this patch with and without the Kubernetes patch.